lib/routes/forklog/index.ts

```diff
@@ -0,0 +1,68 @@
+import { Route } from '@/types';
+import { parseDate } from '@/utils/parse-date';
+import got from '@/utils/got';
+import { load } from 'cheerio';
+import timezone from '@/utils/timezone';
+
+export const route: Route = {
+    path: '/news',
+    categories: ['news', 'crypto'],
+    example: '/forklog',
+    radar: [
+        {
+            source: ['forklog.com/news'],
+            target: '/news',
+        },
+    ],
+    name: 'Forklog - Новости',
```

Comment:
```suggestion
    name: 'Новости',
```
No need to include namespace name in route name.

Reply:
Fixed